PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx

```diff
@@ -564,6 +590,44 @@ bool AliFemtoDreamTrackCuts::PIDCuts(AliFemtoDreamTrack *Track) {
   return pass;
 }
 
+bool AliFemtoDreamTrackCuts::TOFMassCuts(AliFemtoDreamTrack *Track) {
```

Comment:
The function implemented like this makes the PID part very messy.
So we propose as a solution to include this invariant mass TOF cut inside the PIDCuts function.
Please do this by inserting an option (enumerator or bool) to select whether to use TOFnSigma or InvMassTOF.
You can then re-use your function TOFMassCuts and put the lines of the code around lines 545, so where we have the TOF PID part.